src/modules/70-pipeline/pages/execution/ExecutionIACMResourcesView/ExecutionIACMResourcesView.tsx

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { lazy } from 'react'
+import { Container } from '@harness/uicore'
+import ChildAppMounter from 'microfrontends/ChildAppMounter'
+
+/* istanbul ignore next */
+const RemoteIACMApp = lazy(() => import(`iacm/MicroFrontendApp`))
+/* istanbul ignore next */
+const RemoteIACMPipelineResourcesView = lazy(() => import(`iacm/IACMPipelineResources`))
+
+export default function ExecutionIACMResourcesView(): React.ReactElement | null {
```

Comment:
Super minor enhancement, but can we use `ReactNode` instead of `ReactElement | null`? Or better yet, `FC`?
```suggestion
export default function ExecutionIACMResourcesView(): React.ReactNode {
```

Or
```suggestion
export default const ExecutionIACMResourcesView: FC = () => {
```